### PR TITLE
Unit 2.1 Creating HTML Pages

### LINKED ISSUE #5: 2.1: Creating HTML Pages
## Main Goals
1. Create static HTML pages to display user information
2. Create accessible forms to input user information
3. Create and implement a persistent navigation pattern

### When Accomplished
1. The User Profile HTML page
    - [x] has one user profile displaying information following the [user data model](https://github.com/Banno/ux-intern-onboarding/blob/master/resources/specs/localStorage-api-spec.md#the-user-model)
2. The User List HTML page
    - [x] shows several condensed user profiles in contrast to the complete profile on the User Profile page
3. The Create User HTML page
    - [x] contains a form to create a new user with a field for each public property in the [user data model](https://github.com/Banno/ux-intern-onboarding/blob/master/resources/specs/localStorage-api-spec.md#the-user-model)
    - [x] form uses [accessibility markup](http://www.washington.edu/accessibility/checklist/forms/) such as `id` attributes and `<label>` tags to give the form basic accessibility
      - Both the form and submit button do not need to be operational
4. The Edit User HTML page
    - [x] contains a form to edit an existing user
        - Should be identical to the Create User form
    - [x] form uses [accessibility markup](http://www.washington.edu/accessibility/checklist/forms/) such as `id` attributes and `<label>` tags to give the form basic accessibility
        - Both the form and submit button do not need to be operational
5. All HTML pages
  - [x] have an identical and functioning navigation

## Research

- From [Clean Code](https://drive.google.com/a/banno.com/file/d/0Bwz8B78G4U8BLVltUDV4R0VPdTg/view):
  - Chapter 1: Clean Code
  - Chapter 4: Comments
- [Dive into HTML5](http://diveintohtml5.info/semantics.html)
- [HTML & CSS is Hard (But it doesn't have to be)](https://internetingishard.com/html-and-css/)
- From Chad Killingsworth's Tech Talks:
  - [HTML5 as a Semantic Language](https://youtu.be/OJsFiCQ-d1I)
  - [CSS - Quirks of the Box Model](https://youtu.be/sQY7ZLXCZlk)

## Resources
  - [MDN web docs: `<a>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)

## Notes
1. For now, the user profile and user list pages should be populated with dummy data. Eventually this data will be retrieved from `localStorage`.
2. All pages will eventually be tied to the same data, so dummy data should follow [the user data model](https://github.com/Banno/ux-intern-onboarding/blob/master/resources/specs/localStorage-api-spec.md#the-user-model).

### PR DESCRIPTION
**Pull request by frankiebarrios**
## What It Does
This pull request is set to complete Unit 2.1 of the ux-onboarding-project.

This work is taking place to complete unit 2.1 of the ux-onboarding-project.

**Implementation:**
- Added html only structuring to pages required.

[closes #5 ](https://github.com/frankielb/frankie-barrios-ux-aop/issues/5)

## How To Test

N/A

## Notes/Caveats

None
## Checklist

Under penalty of public shaming, I avow that I:
**Many items in list N/A** 
- [X] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [X] Verified that there are no compiler or linter errors or warnings
- [X] Verified the build in production(optimized) mode
- [X] Updated the docs, if necessary
- [X] Included the appropriate labels
- [X] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
**Many items in list N/A** 
- [X] Chrome
- [X] Firefox
- [X] Safari
- [X] Edge
- [X] Internet Explorer

## Dependencies
1. Install Node and npm
2. Install Yarn

### CODE REVIEW

File: html/createUser.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    <h2>Create User</h2>
+
+    <form class="form">
+        <div>
+            <label for="firstName">First Name:</label>
+            <input type="text" name="firstName" id="firstName">
+        </div>
+        <div>
+            <label for="="lastName>Last Name:</label>
```
Comment (collincahill) [inline html/createUser.html]: `for="lastName"`

**Review by collincahill — CHANGES_REQUESTED** [review]
Just the one typo, everything else looks good :smile:

File: html/createUser.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Create User</title>
+</head>
+<body>
+    <ul>
```
Comment (collincahill) [inline html/createUser.html]: This `<ul>` may have a better semantic home within a `<nav>` tag

File: html/createUser.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="userProfile.html">User Profile</a></li>
+        <li><a href="userList.html">User List</a></li>
+        <li><a href="createUser.html">Create User</a></li>
+        <li><a href="editUser.html">Edit User</a></li>
+    </ul>
+
+    <h2>Create User</h2>
```
Comment (collincahill) [inline html/createUser.html]: For the sake of accessibility, HTML pages should usually have one top-level header `h1` tag, which is just the title of the page

File: html/createUser.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    <form class="form">
```
Comment (collincahill) [inline html/createUser.html]: Page content can usually be stored inside of a `<main>` tag

File: html/createUser.html (hunk 3 above)
Comment (collincahill) [reply html/createUser.html]: (You might also put that inside of a `<header>` tag)

**Review by collincahill — COMMENTED** [review]
Here are some additional semantic HTML recommendations

File: html/createUser.html (hunk 2 above)
Comment (frankiebarrios) [reply html/createUser.html]: Thanks for the recommendations. Implemented a lot of these in Unit 2.2 or 2.3 I believe.

**Review by kddrake — APPROVED** [review]

**Pull request merged**